ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,658 @@
+                        breadcrumbsOptions.events.click(button as any, breadcrumbs as any);
+                        // Prevent from click on the current level
+                    } else if (breadcrumbsList[breadcrumbsList.length - 1][1].name !== button.textStr) {
+                        breadcrumbs.multipleDrillUp(breadcrumb[0]);
+                    }
+                })
+                .attr({
+                    padding: 3
+                })
+                .css(breadcrumbsOptions.style as any)
+                .addClass('highcharts-drilldown-breadcrumbs-button')
+                .add(breadcrumbs.breadcrumbsGroup);
+            return button;
+        }
+    }
+
+    /**
+    * Render the separator.
+    *
+    * @requires  modules/drilldown
+    *
+    * @function Highcharts.Chart#renderSeparator
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    * @param {Highcharts.Breadcrumbs} breadcrumb
+    *        Current breadcrumb
+    * @param {Highcharts.Breadcrumbs} posX
+    *        Initial horizontal position
+    * @param {Highcharts.Breadcrumbs} posY
+    *        Initial vertical position
+    * @return {SVGElement|void}
+    *        Returns the SVG button
+    */
+    public renderSeparator(this: Breadcrumbs, breadcrumb: Array<any>, posX: number, posY: number): SVGElement|void {
+        const breadcrumbs = this,
+            chart = this.chart,
+            breadcrumbsOptions = chart.options.drilldown?.breadcrumbs;
+
+        if (breadcrumbsOptions) {
+            return chart.renderer.symbol(
+                breadcrumbsOptions.separator as string,
+                posX + 5,
+                posY - 10,
+                10,
+                10
+            )
+                .attr({
+                    fill: 'white',
+                    stroke: 'black',
+                    'stroke-width': 1,
+                    rotation: 90,
+                    rotationOriginX: posX,
+                    rotationOriginY: posY
+                })
+                .add(breadcrumbs.breadcrumbsGroup)
+                .addClass('highcharts-drilldown-breadcrumbs-separator');
+        }
+    }
+
+    /**
+    * Update.
+    *
+    * @requires  modules/drilldown
+    *
+    * @function Highcharts.Chart#update
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    * @param {Highcharts.Options} userOptions
+    *        Breadcrumbs class.
+    */
+    public update(this: Breadcrumbs, userOptions: Highcharts.Options): void {
+        const chartOptions = H.merge(userOptions, this.options);
+    }
+}
+
+/* eslint-disable no-invalid-this */
+
+if (!H.Breadcrumbs) {
+    H.Breadcrumbs = Breadcrumbs as any;
+
+    addEvent(Chart, 'getMargins', function (e: Chart): void {
+        const chart = this,
+            breadcrumbsOptions = chart.options.drilldown?.breadcrumbs,
+            extraMargin = chart.rangeSelector ? 40 : 30;
```

Comment:
why the value is fixed to 30/40?

Reply:
I changed  `40` into the range selector height.
Second value `30` is a bit more tricky because at that point the breadcrumbs don't exist and why I'm not able to get their height that is why I assumed some default height. I will be able to calculate that height when we have the final design.

Reply:
The Rangeselector is dynamic (responsive design) so you cannot hardcode the height.